src/main/scala/LumberJack.scala

```diff
@@ -16,17 +16,37 @@ object LumberJack extends App
     case firstAryElem =>  firstAryElem foreach{secondAryElem => print(secondAryElem.toString + " ")}
       print('\n')}
 
-  println("")
   var logAmount: Int = 7
+  println("")
+
+  var samllestNum = findsmallestNum(currentPiles)
```

Comment:
you're not using this variable, similar not reassigning value to it, do you need it?